iFixit/react-commerce
# Add item type product lists

closes #354 
fixes #370 

⚠️ After merging this, strapi.ifixit.com needs to be redeployed since the product list type enum has been updated.
The product lists should have `type` set to:
- All parts product list: `all-parts`
- Device parts product list: `parts`
- All tools product list: `all-tools`
- Tools category product list: `tools`
An unset `type` field is assumed as a device parts product list.

cc @sterlinghirsh 

## QA

1. Open [Vercel Preview](https://react-commerce-git-add-item-type-product-lists-ifixit.cominor.com/Parts)
2. Verify than when in device parts product lists you can use the "item type" facet to navigate to the corresponding device item type product list (e.g. iPhone batteries)
3. Verify that the bug described by #370 is fixed
